src/core/objective.jl

```diff
@@ -224,60 +224,6 @@ function _pwl_cost_expression(pm::AbstractPowerModel, x_list, points; nw=0, id=1
 end
 
 
-
-# note that `cost_terms` should be providing in ascending order (the reverse of the Matpower spec.)
-function _polynomial_cost_expression(pm::AbstractPowerModel, x_list::Array{JuMP.VariableRef}, cost_terms; nw=0, id=1, var_name="x")
-    x = sum(x_list)
-    if length(cost_terms) == 0
-        return 0.0
-    elseif length(cost_terms) == 1
-        return cost_terms[1]
-    elseif length(cost_terms) == 2
-        return cost_terms[1] + cost_terms[2]*x
-    elseif length(cost_terms) == 3
-        return cost_terms[1] + cost_terms[2]*x + cost_terms[3]*x^2
-    else # length(cost_terms) >= 4
-        cost_nl = cost_terms[4:end]
-        return JuMP.@expression(pm.model, cost_terms[1] + cost_terms[2]*x + cost_terms[3]*x^2 + sum( v*x^(d+2) for (d,v) in enumerate(cost_nl)) )
-    end
-end

```

Comment:
This method, AFAIK, can just be removed in favor of the other that remains, regardless of what we choose here.

Reply:
Yes, this looks like a work around for an issue that is no longer needed.  It might be worth double checking some of the dependent packages (e.g., PowerModelsAnnex).

If something breaks may need to put back along with a test that checks this control path.